x-pack/plugins/security_solution_ess/public/plugin.ts

```diff
@@ -44,9 +50,13 @@ export class SecuritySolutionEssPlugin
       registerUpsellings(securitySolution.getUpselling(), license, services);
     });
 
-    securitySolution.setComponents({
-      GetStarted: getSecurityGetStartedComponent(services),
-    });
+    securitySolution.setGetStartedPageSettings.setAvailableSteps([
+      OverviewSteps.getToKnowElasticSecurity,
+      AddIntegrationsSteps.connectToDataSources,
+      ViewDashboardSteps.analyzeData,
+      EnablePrebuiltRulesSteps.enablePrebuiltRules,
+      ViewAlertsSteps.viewAlerts,
```

Comment:
Why do we have different enums each defining only one step? Wouldn't make more sense to have only one enum defining all the steps? 🤔

Reply:
Agree, we should simplify the types here. This the low hanging fruit from the previous design when we have multiple items under each steps, I have created an issue to clean this up https://github.com/elastic/kibana/issues/175000